nep-Y.mediawiki

```diff
@@ -288,17 +288,17 @@ Neo-specific codes are documented here, generic JSON-RPC errors like -32602 or
 |-
 | sendrawtransaction  || -500, -501, -502, -503, -504, -505, -506, -507, -508, -509, -510, -511
 |-
-| submitblock         || -500, -501, -508, -509
+| submitblock         || -500, -501, -502, -503, -504, -505, -506, -507, -508, -509, -510, -511
 |-
-| submitoracleresponse|| -602, -603, -604, -605
+| submitoracleresponse|| -508, -602, -603, -604, -605
 |-
 | terminatesession    || -107, -601
 |-
 | traverseiterator    || -107, -108, -601
 |-
 | validateaddress     ||
 |-
-| verifyproof         || -607
+| verifyproof         || -606, -607
```

Comment:
And this one too. I'd just have them logically separated in commits, each modification has its own reason behind and we need to explain the semantics of these changes.

Reply:
Fixed.